snap/snapcraft.yaml

```diff
@@ -28,6 +28,11 @@ apps:
       QT_PLUGIN_PATH: $SNAP/usr/lib/x86_64-linux-gnu/qt5/plugins
     command: bin/multipass
     completer: etc/bash_completion.d/snap.multipass
+  check-kvm-support:
+    command: bin/check_kvm_support
```

Comment:
We don't need to expose this in the snap, it's enough that you install it and then invoke it from a known path.

Reply:
Ok, cool, I'll remove this.

Reply:
Just this, then, and I'll be happy with the code, will test out first thing tomorrow.